operations/power_management/Power_On_and_Start_the_Management_Kubernetes_Cluster.md

```diff
@@ -286,54 +295,31 @@ Verify that the Lustre file system is available from the management cluster.
-    ```
-
-    The pods and containers are normally restored in approximately 10 minutes.
-
-    Because no containers are running, all pods first transition to an `Error` state. The error state indicates that their containers were stopped. The `kubelet` on each node
-    restarts the containers for each pod. The `RESTARTS` column of the `kubectl get pods -A` command increments as each pod progresses through the restart sequence.
-
-    If there are pods in the `MatchNodeSelector` state, delete these pods. Then verify that the pods restart and are in the `Running` state.
-
-1. (`ncn-m001#`) Check the status of the `slurmctld` and `slurmdbd` pods to determine if they are starting:
+    If SDU has been configured on a master node, correct the `collection` link now that its `fuse.s3fs` filesystem is mounted from Ceph storage nodes.
+    Change the link target of `/var/opt/cray/sdu/collection` to `collection-mount` instead to have `collection-local` as the link target. This command checks all master nodes but changes them and restarts `cray-sdu-rda` only if needed.
 
     ```bash
-    kubectl describe pod -n user -lapp=slurmctld
-    kubectl describe pod -n user -lapp=slurmdbd
-    ```
-
-    An error similar to the following may be seen:
-
-    ```text
-    Events:
-      Type     Reason                  Age                    From               Message
-      ----     ------                  ----                   ----               -------
-      Warning  FailedCreatePodSandBox  29m                    kubelet, ncn-w001  Failed to create pod
-    sandbox: rpc error: code = Unknown desc = failed to setup network for sandbox
-    "314ca4285d0706ec3d76a9e953e412d4b0712da4d0cb8138162b53d807d07491": Multus: Err in tearing down failed
-    plugins: Multus: error in invoke Delegate add - "macvlan": failed to allocate for range 0: no IP addresses available in range set: 10.252.2.4-10.252.2.4
-      Warning  FailedCreatePodSandBox  29m                    kubelet, ncn-w001  Failed to create pod
-    sandbox: rpc error: code = Unknown desc = failed to setup network for sandbox
+    pdsh -w ncn-m00[1-3]  '(cd /var/opt/cray/sdu; if [ -L "collection" ]; then if [ "$(readlink collection)" = "collection-local" ]; then rm collection; ln -s collection-mount collection; systemctl restart cray-sdu-rda; fi; fi)'
     ```
 
-    If the preceding error is displayed, then remove all files in the following directories on all worker nodes:
+1. (`ncn-m001#`) Check that `spire` pods have started.
 
-    * `/var/lib/cni/networks/macvlan-slurmctld-nmn-conf`
-    * `/var/lib/cni/networks/macvlan-slurmdbd-nmn-conf`
+    **Note:** Because no containers are running, all pods first transition to an `Error` state. The `Error` state indicates that their containers were stopped. The `kubelet` on each node
+    restarts the containers for each pod. The `RESTARTS` column of the `kubectl get pods -A` command increments as each pod progresses through the restart sequence.
```

Comment:
Did you intend to copy this note here from the section about generally monitoring Kubernetes pod status? It doesn't really flow right here, IMO.

Reply:
fixed